evap/evaluation/tests/test_models.py

```diff
@@ -222,3 +223,49 @@ def test_archiving_doesnt_change_single_results_participant_count(self):
         course._archive()
         self.assertEqual(course._participant_count, 5)
         self.assertEqual(course._voter_count, 5)
+
+
+class TestEmails(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.other_user = mommy.make(UserProfile, email="other@extern.com")
+        cls.user = mommy.make(UserProfile, email="extern@extern.com")
+        cls.user.generate_login_key()
+
+        cls.course = mommy.make(Course)
+        contribution = mommy.make(Contribution, course=cls.course, contributor=cls.user, responsible=True)
+
+        cls.template = mommy.make(EmailTemplate, body="{{ login_url }}")
+
+        EmailTemplate.objects.filter(name="Login Key Created").update(body="{{ user.login_url }}")
+
+    def test_no_login_url_when_delegates_in_cc(self):
+        self.user.delegates.add(self.other_user)
+        self.user.save()
+        EmailTemplate.send_to_users_in_courses(self.template, [self.course], "contributors", use_cc=True)
+        self.assertEqual(len(mail.outbox), 2)
+        self.assertFalse("userkey" in mail.outbox[0].body)  # message does not contain the login url
+        self.assertTrue("userkey" in mail.outbox[1].body)  # separate email with login url was sent
+
+    def test_no_login_url_when_cc_users_in_cc(self):
+        self.user.cc_users.add(self.other_user)
+        self.user.save()
+        EmailTemplate.send_to_users_in_courses(self.template, [self.course], "contributors", use_cc=True)
+        self.assertEqual(len(mail.outbox), 2)
+        self.assertFalse("userkey" in mail.outbox[0].body)  # message does not contain the login url
+        self.assertTrue("userkey" in mail.outbox[1].body)  # separate email with login url was sent
```

Comment:
maybe assert that the second email is sent to the right person and no-one is in cc?